src/js/components/Text/text.stories.js

```diff
@@ -30,6 +32,22 @@ const Color = () => (
   </Grommet>
 );
 
+const WordBreak = () => (
+  <Grommet theme={grommet}>
+    {wordBreakValues.map(value => (
+      <Box key={value} margin="small">
+        <Heading level={4}>{`word-break: ${value};`}</Heading>
+        <Text wordBreak={value}>
```

Comment:
Something doesn't look right to me with the way the example is rendered, The Text is rendered the same no matter what the value is:

![Screen Shot 2019-04-29 at 12 42 05 PM](https://user-images.githubusercontent.com/6320236/56919334-8cf78a00-6a7d-11e9-9c24-42c748aa3f43.png)

I also looked into the CSS, and except `word-break: normal` the other Texts do not have the `word-break` prop at all. After I added it manually, it did make an impact on the way the lines renders as follows:

![Screen Shot 2019-04-29 at 12 50 33 PM](https://user-images.githubusercontent.com/6320236/56919485-ea8bd680-6a7d-11e9-832c-8054b6e587bb.png)

Reply:
probably my fault here. i suggested to use breakAll instead of break-all so in order for this to work we would need a map. something like https://github.com/grommet/grommet/blob/master/src/js/components/Box/StyledBox.js#L136-L143 
@ericsoderberghp what do you feel on this? should we keep normal css property here (break-all) or add a map ?

Reply:
I took at look at our Box property values. Animation uses camelCase for `animation` but dashes for `direction`, and `round`. Layer and Stack also use dashes. Given that, my inclination is to stick with the CSS values on this one, with dashes, since the CSS values are simple enough. I realize we don't always use CSS values, such as `align="start"` instead of `align="flex-start"` because we bias towards non-CSS savvy users and prefer simpler value strings, unless the CSS values are simple enough.

Reply:
@ShimiSun You're right here. Looks like I messed up while trying to convert from kebab case to camel case. I'll get to this.